src/components/Auth/AuthBody.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { FC, memo } from 'react';
+
+import example from 'example-pathway.png';
+import github from 'github.png';
+import email from 'email.png';
+import useStyles from './styles';
+
+const AuthBody: FC = () => {
+  const styles = useStyles();
+  return (
+    <>
+      <img src={example} alt="pathway" className={styles.exampleImage} />
+      <label className={styles.mainText}>A simple app to build</label>
+      <br />
+      <label className={styles.mainText}> customized clinical pathways</label>
+      <br />
+      <label className={styles.subText}>using mCODE. a standardized data </label>
```

Comment:
This should be a comma
```suggestion
      <label className={styles.subText}>using mCODE, a standardized data </label>
```